meiki_ui/src/components/app/DeleteModal.svelte

```diff
@@ -1,15 +1,15 @@
 <script lang="ts">
     import { createEventDispatcher } from "svelte"
     import { deleteNote } from "$lib/api/notes"
-    import currentNote from "$lib/stores/currentNote"
     import ConfirmModal from "$cmp/modal/ConfirmModal.svelte"
 
+    export let noteID: string
     export let error: string = ""
     const dispatchEvent = createEventDispatcher()
 
     async function deleteCurrentNote() {
         try {
-            await deleteNote($currentNote)
+            await deleteNote(noteID)
```

Comment:
Is it better to pass noteID as an argument rather than picking it up from scope?